miniconda/Dockerfile

```diff
@@ -3,8 +3,9 @@ FROM debian:7.4
 MAINTAINER Travis Swicegood
 
 RUN apt-get update && apt-get install -y wget bzip2
-RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh
-RUN wget --quiet http://repo.continuum.io/miniconda/Miniconda3-3.6.0-Linux-x86_64.sh && \
-    /bin/bash /Miniconda3-3.6.0-Linux-x86_64.sh -b -p /opt/anaconda && \
-    rm Miniconda3-3.6.0-Linux-x86_64.sh
+RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh && \
+    wget --quiet http://repo.continuum.io/miniconda/Miniconda-3.7.3-Linux-x86_64.sh && \
+    /bin/bash /Miniconda-3.7.3-Linux-x86_64.sh -b -p /opt/anaconda && \
```

Comment:
As a miniconda user :wink:, why putting it in `opt/anaconda` if there is not anaconda there? I don't know enough about the internal of this PR/implementation, so... sorry if the question is silly... I just saw this in my daily issues revision and I am wondering... thanks!